packages/config-loader/src/loader.test.ts

```diff
@@ -145,25 +145,4 @@ describe('loadConfig', () => {
       },
     ]);
   });
-
-  it('coerces port to a number', async () => {
```

Comment:
Removed this since coercion is done at read-time instead of when loading

Reply:
Reverted the commit that introduced this test

Reply:
👍